README.md

```diff
@@ -54,40 +54,83 @@ Documentation for ert is located at [https://ert.readthedocs.io/en/latest/](http
 
 ## Developing
 
-*ERT* is Python and C software. To start developing, install it in editable
-mode:
+*ERT* uses Python for user-facing code and C++ for some backend code. Python is
+the easiest to work with and is likely what most developers will work with.
 
-```
-$ git clone https://github.com/equinor/ert
-$ cd ert
-$ pip install -e .
+### Developing Python
+
+To start developing the Python code, install ERT in editable mode into a virtualenv:
+
+```sh
+# Create and enable a virtualenv
+python3 -m venv my_virtualenv
+source my_virtualenv/bin/activate
+
+# Update build dependencies
+pip install --upgrade pip wheel setuptools
+
+# Download and install ERT
+git clone https://github.com/equinor/ert
+cd ert
+pip install --editable .
 ```
 
 Additional development packages must be installed to run the test suite:
-```
-$ pip install -r dev-requirements.txt
-$ pytest tests/
+```sh
+pip install -r dev-requirements.txt
+pytest tests/
 ```
 
-<strong>For Mac-users</strong> <em>The default maximum number of open files is normally relatively low on MacOS.
-This is likely to make tests crash with mysterious error-messages.
-You can inspect the current limits in your shell by issuing he command 'ulimit -a'.
-In order to increase maximum number of open files, run 'ulimit -n 16384' (or some other large number)
-and put the command in your .profile to make it persist.
-</em>
+Since we've installed ERT in `--editable` mode, it is now possible to modify the
+Python code without reinstalling. Simply restart ERT after making your changes.
 
-ERT is meant to be installed using `setup.py`, directly or using `pip
-install ./`. The `CMakeLists.txt` in libres exists, but is used by `setup.py`
-to generate the ERT C library (the C library formerly known as *libres*) and
-by Github Actions to run C tests.
+### Developing C++
```

Comment:
I think this should be the only option an renamed to `Developing`

Reply:
I have split the `Developing` section into two parts, one for Python and one for C++. I reckon the majority of developers will only need the Python part so it's uncluttered. The main difference is the bootstrapping, ie. `pip install -e .` vs `script/build setup`. But if you want I can combine them into a single section.